easybuild/easyconfigs/b/boost_histogram/boost_histogram-1.2.1-foss-2021a.eb

```
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'boost_histogram'
+version = '1.2.1'
+
+homepage = 'https://boost-histogram.readthedocs.io'
+description = """Boost-histogram is a Python package providing Python bindings for Boost.Histogram."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['a27842b2f1cfecc509382da2b25b03056354696482b38ec3c0220af0fc9b7579']
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('pybind11', '2.6.2'),
+]
+
+use_pip = True
```

You'll also need to enable `download_dep_fail = True` here, now that you're using `PythonPackage` as easyblock:

```suggestion
download_dep_fail = True
use_pip = True
```

---

I did it. It works?